cluster-logging/eo-sub.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  generateName: "elasticsearch-"
+  namespace: "openshift-operators-redhat" 
+spec:
+  channel: "4.2" 
+  installPlanApproval: "Automatic"
+  source: "redhat-operators"
+  sourceNamespace: "openshift-marketplace"
+  name: "elasticsearch-operator"
```

Comment:
I'm afraid I have no idea, but I'll check which version was used.

Reply:
it's the `4.2.13-201912230557` version which is installed, that is, the version displayed in the `OperatorHub` section of the web console, but I'm not sure this answers your question :/

Reply:
Yeah, I would expect that it could take the latest one... I believe that we can remove the https://github.com/codeready-toolchain/toolchain-infra/pull/6/files#diff-127dc482a4e876254e0c51a147d849f0R13 to be consistent and to let OLM pick the latest version every time

Reply:
I'm sorry, I can't see what you wanted to point at with the link. Would you mind giving more details here in the comments?

Reply:
I was referring to the line 12 
```
  startingCSV: clusterlogging.4.2.13-201912230557
```
in `cluster-logging/clo-sub.yaml`  file. Do we need to specify the startingCSV if the OLM pick the latest one anyway?

Reply:
see my suggestion: https://github.com/codeready-toolchain/toolchain-infra/pull/6#discussion_r366971526